chans/chans.go

```diff
@@ -32,6 +49,22 @@ func Filter[T any](source <-chan T, predicate func(T) bool) <-chan T {
 	return out
 }
 
+// Flatten flattens a chan of chans into a chan of elements
+func Flatten[T any](source <-chan <-chan T) <-chan T {
+	out := make(chan T, cap(source))
```

Comment:
```suggestion
	out := make(chan T)
```
I think it's better to keep it unbuffered, this way, the flattened channel effectively has the same buffer as the sum of the original channels

Reply:
agree, I didn't change it from the original implementation, but unbuffered would be my choice too